extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchAggregatorTest.java

```diff
@@ -98,6 +110,12 @@ public static Collection<?> constructorFeeder()
     return constructors;
   }
 
+  @After
+  public void teardown() throws IOException
+  {
+    closer.close();
+  }
+
```

Comment:
This closer is used to unmap a segment, such behaviors tend to be relatively expensive and I'm scared that the pattern in this test will lead to more tests that build and kill the same segment multiple times, which has been a common pattern that slows down our tests.

Would it make sense to have these be at `BeforeClass` and `AfterClass` and build the segment once such that it could be reused by all of the tests on the class?  For this one case, that would amount to reuse for all of the different parameters of the test, which I think would be a good thing.

Reply:
yeah, you caught me being pretty lazy to write these tests in the easiest/fastest way i could come up with, will look into improving this